13/umbraco-cms/fundamentals/setup/upgrading/README.md

```diff
@@ -102,7 +102,7 @@ Add a package reference to your project by executing the `dotnet add package Umb
 Run `dotnet restore` to install the package.
 
 {% hint style="warning" %}
-If you are using SQL CE in your project you will need to run `dotnet add package Umbraco.Cms.SqlCe --version <VERSION>` too before running the `dotnet restore` command.
+If you are using SQL CE in your project you will need to run `dotnet add package Umbraco.Cms.Persistence.Sqlite --version <VERSION>` too before running the `dotnet restore` command.
 {% endhint %}
```

Comment:
This whole hint should be removed in this case, as it refers specifically to SQL CE. From v10 SQL CE is not used anymore: https://umbraco.com/products/umbraco-cms/umbraco-10/#:~:text=This%20replaces%20SQL%20CE%20which%20has%20reached%20End%20Of%20Life.%C2%A0